examples/tests/invalid/cloud_function_expects_inflight.w

```diff
@@ -0,0 +1,6 @@
+bring cloud;
+
+new cloud.Function((name: str): str => {
+    return "Hello ${name}";
+});
+// ^ Expected type to be "inflight (any): any", but got "preflight (str): str" instead
```

Comment:
I feel like `preflight (str): str` could be confusing (since isn't preflight the defaul" phase in a Wing file?)? So maybe the error should be:

```
Expected type to be "inflight (any): any", but got "(str): str" instead
```

But this opens the question of how do we indicate a phase-independent function? `anyflight (str): str` maybe?

Curious what other people think

Reply:
I'm tempted to say the overly-verbose `preflight (str): str` in diagnostics is better as a starting point since it reinforces the idea of preflight vs inflight

Reply:
I think keeping `preflight` in the type is good, especially in the context of a diagnostic. It helps to make the difference stick out more. Plus, general consistency is nice.


For phase independent, I think excluding any keyword is ok but I think having one would be great too.

```
Expected type to be "independent (any): any", but got "preflight (str): str" instead
```

I mean it looks pretty verbose but I wouldn't call it confusing. Not sure about `independent` of course, we should probably throw around more options for that.

Reply:
Let's try without the extra keyword and see how it goes